test/testcafe/spec/Identify_spec.js

```diff
@@ -310,7 +310,10 @@ test.requestHooks(identifyRequestLogger, identifyMockWithFingerprintError)('shou
 test.requestHooks(identifyRequestLogger, baseIdentifyMock)('should pre-populate identifier field with username config', async t => {
   const identityPage = await setup(t);
   await rerenderWidget({
-    username: 'myTestUsername@okta.com'
+    username: 'myTestUsername@okta.com',
+    features: { // Ensure remember my username feature is off so it doesn't override config
+      rememberMe: false,
```

Comment:
You don't need this because it's undefined by default which is falsey, and updating this test means the existing functionality has changed, which it hasn't?

Reply:
https://github.com/okta/okta-signin-widget#feature-flags - we're leveraging the same flag that was there before which by default is true, it's also set to true as part of our default .widgetrc file.

Reply:
ah didn't know it defaulted to true